celery_utils/logged_task.py

```diff
@@ -36,6 +36,14 @@ def on_retry(self, exc, task_id, args, kwargs, einfo):
         """
         Capture the exception that caused the task to be retried, if any.
         """
-        super(LoggedTask, self).on_retry(exc, task_id, args, kwargs, einfo)
         if exc is not None:
             log.warning('[{}] retried due to {}'.format(task_id, einfo.traceback))
+        super(LoggedTask, self).on_retry(exc, task_id, args, kwargs, einfo)
+
+    def on_failure(self, exc, task_id, args, kwargs, einfo):
```

Comment:
It should be possible to set up the python logger so that info level messages from celery are sent through (but aren't sent through for other loggers). But we might have to overhaul the edx-platform logging configuration to do that (which is in itself long overdue).